.github/workflows/github-release.yml

```diff
@@ -0,0 +1,35 @@
+# This workflow will create a release when there is a push on the main branch
+# For more information see: https://github.com/marvinpinto/action-automatic-releases
+
+name: GCP Scanner Auto Release
+
+on:
+  push:
+    branches: [ "main" ]
+    tags:
+      - "v*"
+
+jobs:
+  tagged-release:
+    name: "Tagged Release"
+    runs-on: "ubuntu-latest"
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v2
```

Comment:
We should use `v3` instead of `v2` (this comes as a warning in the GitHub Actions tab)
This is because node12 is deprecated. [refer this [link](https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/) and this for [changelog](https://github.com/actions/checkout/blob/main/CHANGELOG.md#v300)]
```suggestion
      - uses: actions/checkout@v3
      - name: Set up Python 3.10
        uses: actions/setup-python@v3
```